cloud-sql/sql-server/Startup.cs

```diff
@@ -152,6 +152,7 @@ public static SqlConnectionStringBuilder GetSqlServerConnectionString()
             // connection with the server no longer exists.
             // [END cloud_sql_sqlserver_dotnet_ado_lifetime]
             // [END_EXCLUDE]
+            connectionString.TrustServerCertificate = true;

```

Comment:
I'm a little confused by all the tags here, just checking that this won't appear on samples? I don't think we should be recommending this setting by default.

Reply:
Updated as suggested to exclude the additional setting so that it doesn't appear in documentation code snippets.